src/main/java/mate/academy/controllers/UserController.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.controllers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.dto.UserResponseDto;
+import mate.academy.model.User;
+import mate.academy.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/inject")
+    @ResponseBody
```

Comment:
do we need ResponseBody annotation here?

Reply:
No, of course)